retsinformation.js

```diff
@@ -0,0 +1,182 @@
+{
+	"translatorID": "feef66bf-4b52-498f-a586-8e9a99dc07a0",
+	"label": "Retsinformation",
+	"creator": "Roald Frøsig",
+	"target": "^https?://(?:www\\.)?retsinformation\\.dk/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "g",
+	"lastUpdated": "2016-01-27 14:38:37"
+}
+
```

Comment:
You should add the AGPL licence explicitely here, see for example my block for that here: https://github.com/zuphilip/translators/wiki/Common-code-blocks-for-translators#licence-block